src/components/toast/ToastContext.tsx

```diff
@@ -62,12 +52,14 @@ export const ToastContextProvider = ({ children }: Props) => {
         <div className="toast-container">
           <div className="toasts-wrapper">
             <AnimatePresence>
-              {toasts.map(({ toast, props }) => {
+              {toasts.map(({ toast, props, uuid }) => {
+                const key = `toast-${toast}-${uuid}`;
                 return (
                   <Toast
-                    toastKey={`toast-${toast}-${Date.now()}`}
-                    key={`toast-${toast}-${Date.now()}`}
+                    toastKey={key}
+                    key={key}
                     intent={props?.intent}
+                    data-dog-id={key}
```

Comment:
I think this is unnecessary 🤔 
```suggestion
                    intent={props?.intent}
```

Reply:
whoops